File: fizzbuzz.py
from time import time
from tqdm import tqdm

start = time()
final = ['FizzBuzz' if i%15==0 else 'Buzz' if i%3==0 else 'Fizz' if i%5==0 else str(i) for i in tqdm(range(1,101))]
for i in final:
    print(i)
duration = time() - start
print(f'Done! Took \u001b[92m{duration}\u001b[0m seconds.')

File: fizzbuzz.c
#include <stdio.h>
#include <stdbool.h>

int main()
{
    for(int i = 1; i < 101; i++) {
        bool divByThree = i % 3 == 0;
        bool divByFive = i % 5 == 0;
        if (!divByThree && !divByFive) {
            printf("%d", i);
        } else {
            if (divByThree) {
                printf("Fizz");
            }
            if (divByFive) {
                printf("Buzz");
            }
        }
        printf("\n");
    }

    return 0;
}
